x/slashing/abci_test.go

```diff
@@ -9,22 +9,23 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/internal/keeper"
```

Comment:
no need for an import alias here.

Reply:
this is mostly for util test funcs that imho shouldn't be aliased